.github/workflows/test.yml

```diff
@@ -243,7 +249,8 @@ jobs:
 
       - name: Generate
         run: |
-          go run github.com/steebchen/prisma-client-go db push
+          go run github.com/steebchen/prisma-client-go migrate deploy
+          task generate-all
```

Comment:
@abelanger5 why does the test action need this now? the idea was that this just runs unit test which doesn't need all the generated stuff to keep it small and simple

Reply:
This is the load test job - we at least need to run `generate-go`, but perhaps we don't need to run the other generators?

Reply:
ah my mistake. probably fine then